src/tactics.ml

```diff
@@ -103,6 +103,41 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
       handle_refine (Rewrite.reflexivity tac.pos ps)
   | P_tac_sym           ->
       handle_refine (Rewrite.symmetry tac.pos ps)
+  | P_tac_why3(s)    ->
+      (* get the goal to prove *)
+      let (hypotheses, trm) = Proof.Goal.get_type g in
+      (* get the default or the indicated name of the prover. *)
+      let prover_name =
+        match s with
+        | None          -> Timed.(!Why3prover.default_prover)
+        | Some(name)    -> name
+      in
+      (* translate from lambdapi to why3 terms. *)
+      let (constants_table, hyps, why3term) =
+          Why3prop.translate tac.pos ps.proof_builtins (hypotheses, trm) in
```

Comment:
`in` on its own line.

Reply:
Fixed in 7be0a7ca2d7a99bf157dc5b6089886bba4d48076